src/XMakeTasks/UnitTests/WriteCodeFragment_Tests.cs

```diff
@@ -530,6 +530,36 @@ public void OneAttributePositionalAndNamedParams()
 
             File.Delete(task.OutputFile.ItemSpec);
         }
+
+        /// <summary>
+        /// Some attributes only allow positional constructor arguments.
+        /// To set those, use metadata names like "_Parameter1", "_Parameter2" etc.
+        /// These can also be combined with named params.
+        /// </summary>
+        [Fact]
+        public void OneAttributePositionalAndNamedParamsVisualBasic()
```

Comment:
Wouldn't hurt, but in the regular case it would really just test the CodeDom not any code in MSBuild. This scenario helped here because I wanted to make sure VB named parameters worked and I didn't have any examples.